pkg/multichannel/multichannel.go

```diff
@@ -0,0 +1,59 @@
+package multichannel
```

Comment:
any reason this is not in the existing `license` package?

Reply:
ah - originally had more in here and ended up with some circular imports when i had it under the `license` package but its pretty small now and the only other pkg import is replicatedapp.

Reply:
relocated in:  https://github.com/replicatedhq/kots/pull/4767/commits/61ea5d54becda901b8e5e86a0a67f8569d7b5f45